apps/benchmark/src/implementations/inline-styles/Dot.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { DotProps } from '../types';
+
+const Dot: React.FC<DotProps> = ({ size, x, y, children, color }) => (
+  <div
+    style={{
+      ...styles.root,
+      ...{
+        borderBottomColor: color,
+        borderRightWidth: `${size / 2}px`,
+        borderBottomWidth: `${size / 2}px`,
+        borderLeftWidth: `${size / 2}px`,
+        marginLeft: `${x}px`,
+        marginTop: `${y}px`,
+      },
```

Comment:
```suggestion
```

Reply:
Done in [4ed98dcb](https://github.com/microsoft/griffel/pull/132/commits/4ed98dcbe09cfc92ebbf8ba627bcbff2301fa426)